addon/components/validated-input/types/checkbox-group.js

```diff
@@ -0,0 +1,23 @@
+import Component from "@ember/component";
+import { action } from "@ember/object";
+
+import layout from "../../../templates/components/validated-input/types/checkbox-group";
+
+export default Component.extend({
```

Comment:
Is there a specific reason why you are still generating non-glimmer components or is this just from the blueprint?

Reply:
Same as above, just to be consistent with the other components. I agree though, we should refactor all of them (in a separate PR).